src/components/shared/partials/FormInputs.jsx

```diff
@@ -0,0 +1,43 @@
+import { useState } from 'react';
+//import { AuthContext } from '../../../context/AuthContext';
+//import { STAGES, currentUser } from '../../../data/formInformation';
+//import { count } from 'firebase/firestore';
+
+export default function Inputs({ className, formData, onChange, validated }) {
+	const { type, minLength, maxLength, regex, id, placeholder, value } = formData;
+	const [errorMessage, setErrorMessage] = useState('');
+
+	const validateInput = event => {
+		const value = event.target.value;
+		if (regex && value) {
+			const isValid = value.match(regex);
+			if (!isValid) {
+				console.log(`Invalid input for ${id}`);
```

Comment:
NP: remove console.log

Reply:
done. had written for testing purpose. removed in last commit.

Reply:
were there for testing. removed in last commit.